### PR TITLE
Add Blockstream Jade support

### PR DESCRIPTION
**Pull request by ben-kaufman**
Add support for Blockstream Jade (single sig).
I had to make a few small changes/ hacks in the files I got from the HWI Jade PR to make this work, but tested with signet and everything works fine now for me.

### CODE REVIEW

File: src/cryptoadvance/specter/device.py [hunk 1]
```diff
@@ -124,6 +124,8 @@ def key_types(self, network="main"):
 
     def has_key_types(self, wallet_type, network="main"):
         if wallet_type == "multisig":
+            if self.device_type == "jade":
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/device.py:127]: I don't think it's a good idea to do jade- specific things in global device class.
You can overwrite `has_key_types` method in jade class.

**Review by stepansnigirev — CHANGES_REQUESTED** [review]
Please move jade-specific code to jade device class.
Also don't we need to add udev rules for Linux?

https://github.com/JamieDriver/HWI/blob/blockstream_jade_support/hwilib/udev/55-usb-jade.rules

File: src/cryptoadvance/specter/device.py (hunk 1 above)
Comment (ben-kaufman) [reply src/cryptoadvance/specter/device.py]: Makes sense, moved it.

File: src/cryptoadvance/specter/devices/jade.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class Jade(HWIDevice):
+    device_type = "jade"
+    name = "Jade"
+    icon = "jade_icon.svg"
+
+    supports_hwi_toggle_passphrase = True
+    supports_hwi_multisig_display_address = True
+
+    def __init__(self, name, alias, keys, blinding_key, fullpath, manager):
+        HWIDevice.__init__(self, name, alias, keys, blinding_key, fullpath, manager)
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/devices/jade.py:14]: If you are using exactly the same function as the parent class you can remove the whole `__init__` function declaration

File: src/cryptoadvance/specter/hwi_rpc.py [hunk 3]
```diff
@@ -417,6 +418,9 @@ def _get_client(
                 if chain == "liquidv1":
                     chain = "main"
                 client.chain = Chain.argparse(chain)
+                if type(client) is JadeClient:
+                    if chain == "signet":
+                        client.chain = Chain.TEST
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/hwi_rpc.py:423]: If Jade requires a replacement of signet to testnet, does it also apply to regtest or it works fine with regtest?

**Review by stepansnigirev — COMMENTED** [review]
Thanks for the changes.
Is there a reason you deleted `udev/README.md` completely?

File: src/cryptoadvance/specter/hwi_rpc.py (hunk 3 above)
Comment (ben-kaufman) [reply src/cryptoadvance/specter/hwi_rpc.py:423]: No regtest works fine

**Comment by ben-kaufman** [comment]
No it was by mistake, added it back now

**Review by stepansnigirev — APPROVED** [review]

**Pull request merged**